fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py

```diff
@@ -991,6 +996,8 @@ def encode_list(
             pad_type_id (Optional[int], optional): _description_. Defaults to 0. (TODO: raise exception)
             return_overflow_info (Optional[bool], optional): _description_. If True return an additional string with overflow information. Defaults to False.
             on_unknown: (Optional[str], optional): What happens if unknown tokens (i.e. ones mapped to <UNK>) are encountered: 'raise' or 'warn'
+            verbose (Optional[int], optional): verbosity level. 0,1: warning notification, 2: warning with partial data, 3: warning
```

Comment:
I suggest this be spaced out a little, to allow values in-between,

Reply:
I actually do see a need for in-between here:  For our use on gene expression, we know we will have some unknown tokens (it's pretty inherent to the data), so we can't use 2 or 3, as they will flood the log files, but we don't want to ignore it all together as we may have some wrongly unmatched tokens.  I suspect we need something like "report each new un-mapped token", or better yet, give a report on the unmapped tokens found at the end (of each epoch?)

Reply:
This brings us back to the problem of finding which tokens map to UNK. In any case, things like new unmapped tokens and epoch statistics do not fall under tokenizer jurisdiction. This is something you should do outside of it.

Reply:
we can probably manage by pulling the encoded object from the tokenizer via "key_out_tokenized_object" and extracting the problematic tokens from that.  
so - 0 as "no warning", default is 1?